bundles/org.openhab.binding.insteon2/README.md

```diff
@@ -0,0 +1,941 @@
+# Insteon Binding
+
+Insteon is a proprietary home automation system that enables light switches, lights, thermostats, leak sensors, remote controls, motion sensors, and other electrically powered devices to interoperate through power lines, radio frequency (RF) communications, or both (dual-band)
+More about Insteon can be found on [Wikipedia](https://en.wikipedia.org/wiki/Insteon).
+
+This binding is mostly a rewrite of the [legacy binding](https://www.openhab.org/addons/bindings/insteon/) adding some much needed improvements such as the automated device configuration layer.
+It provides access to the Insteon network by means of either an Insteon PowerLinc Modem (PLM), a legacy Insteon Hub 2242-222 or the current 2245-222 Insteon Hub.
+The modem can be connected to the openHAB server either via a serial port (Model 2413S) or a USB port (Model 2413U).
+The Insteon PowerLinc Controller (Model 2414U) is not supported since it is a PLC not a PLM.
+The modem can also be connected via TCP (such as ser2net).
+The binding translates openHAB commands into Insteon messages and sends them on the Insteon network.
+Relevant messages from the Insteon network (like notifications about switches being toggled) are picked up by the modem and converted to openHAB state updates by the binding.
+The binding also supports sending and receiving of legacy X10 messages.
+
+The openHAB binding supports configuring most of the device local settings, linking a device to the modem, managing link database records and scenes along with monitoring inbound/outbound messages.
+Other tools can be used to managed Insteon devices, such as the [Insteon Terminal](https://github.com/pfrommerd/insteon-terminal) open source project, or the [HouseLinc](https://www.insteon.com/houselinc) software from Insteon can also be used for configuration, but it wipes the modem link database clean on its initial use, requiring to re-link the modem to all devices.
+
+At startup, the binding will download the modem database along with each configured device all-link database if not previously downloaded and currently awake.
+Therefore, the initialization on the first start may take some additional time to complete depending on the number of devices configured.
+The modem and device link databases are only downloaded once unless the binding receives an indication that a database was updated or marked to be refreshed via the [openHAB console](#console-commands).
+
+If switching from the legacy binding, there is unfortunately no direct path to convert existing configuration due to the new way bridges, devices and scenes are configured and discovered.
+It is recommended to disable the legacy network bridge to prevent both bindings from connecting to the same modem at the same time while keeping the ability to port some of the configuration over.
```

Comment:
```suggestion
It is recommended to disable the Insteon network bridge to prevent both bindings from connecting to the same modem at the same time while keeping the ability to port some of the configuration over.
```